api/cpp/tests/platform_eventloop.cpp

```diff
@@ -39,7 +42,7 @@ struct TestPlatform : slint_platform::Platform
                         quit = false;
                         break;
                     }
-                    cv.wait(lock);
+                    cv.wait_for(lock, slint_platform::duration_until_next_timer_update());
```

Comment:
Like this would be `slint_platform::duration_until_next_timer_update().value_or(std::chrono::milliseconds::max())`, which I think is more expressive.